spec/requests/mc_responses_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe "McResponses", type: :request do
+
+  describe "positive tests" do 
+
+    let!(:survey) {Survey.create(title: "test", id: 1, isLive: false, wentLiveAt: nil)}
+    let!(:mc_question_test) {McQuestion.create(question: "test question", survey_id: survey.id, id: 1), mc_options: {options: ["op1", "op2"]}}
+    # write in a line here that makes the survey live so that we can add in a survey responder and text response
+    # let!(:survey) {Survey.update(isLive: true)}
+    let!(:surveyResponder) {SurveyResponder.create(survey_id: survey.id)}
+
+    #------------------------------- YOU STOPPED HERE CONTINUE FINISHING POSITIVE AND NEGATIVE TESTS FOR THIS-------------------------------------
```

Comment:
these look like left over comments that may not be needed anymore, we should remove them!